source/socketstuff.lua

```diff
@@ -191,7 +192,8 @@ function socketstuff.startHosting(myServerPort)
     udpHost = socket.udp()
     udpHost:settimeout(0)
     udpHost:setsockname('*', myServerPort)
-    print("Server started on port " .. myServerPort)
+	local sockname = udpHost:getsockname()

```

Comment:
Am I too nitpicky? * *self-doubts arise* * `sockName`

Reply:
Yep. Should be camel.

Was getting annoyed with sockets at this point.

About to rewrite the whole lot with enet.